src/slurmmail/cli.py

```diff
@@ -891,6 +898,11 @@ def send_mail_main():
         options.email_from_name = config.get(section, "emailFromName")
         options.email_subject = config.get(section, "emailSubject")
 
+        if config.has_option(section, "emailHeaders"):
+            email_headers = [ x.strip().split(":", maxsplit=1) for x in config.get(section, "emailHeaders").split(";") ]
+            for email_header in email_headers:
+                options.email_headers[email_header[0].strip()] = email_header[1].strip()
```

Comment:
We also need to ensure that the user had not provided e-mail headers that are set by Slurm-Mail. These are:

- `To`
- `From`
- `Date`
- `Message-ID`

Otherwise these will get overwritten. Perhaps an array of protected headers could be added which are then checked against when processing the `slurm.conf` file?